guide/how-it-works/liquidity.md

```diff
@@ -0,0 +1,314 @@
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/liquidity/liquidity-07.png"
+   retina = "/assets/images/guide/how-it-works/liquidity/liquidity-07@2x.png"
+   mobile = "/assets/images/guide/how-it-works/liquidity/liquidity-07-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/liquidity/liquidity-07-mobile@2x.png"
+   alt-text = "Rebalancing channels on the lightning network"
+   width = 1200
+   height = 800
+   layout = "full-width"
+%}
+
+While the example thus far has focused on routing nodes, mobile nodes also fit into this scenario. Now that our node operator Lori has maintained a well-connected routing node, she can use her node to extend liquidity to mobile nodes.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/liquidity/liquidity-08.png"
+   retina = "/assets/images/guide/how-it-works/liquidity/liquidity-08@2x.png"
+   mobile = "/assets/images/guide/how-it-works/liquidity/liquidity-08-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/liquidity/liquidity-08-mobile@2x.png"
+   alt-text = "Rebalancing channels on the lightning network"
+   width = 1200
+   height = 800
+   layout = "full-width"
+%}
+
+## Handling liquidity for wallet users
+
+The above examples were designed to help you understand liquidity on a conceptual level. Ideally, the user of a mobile [daily spending wallet]({{'/guide/daily-spending-wallet/' | relative_url}}) never has to consider any of the liquidity sourcing that happens behind the scenes. Consider how you might design a bitcoin product that doesn't require the user to understand lightning channel management.
+
+### Inbound liquidity
+
+Helping the user get inbound liquidity is probably more important than outbound liquidity for most use cases. Without inbound liquidity, the user can not receive bitcoin over lightning. This makes it very difficult for a new bitcoin user to receive their first bitcoin.
+
+If the user does not have bitcoin in their wallet, they do not need outbound liquidity. On the other hand, if the user has inbound liquidity, then they are free to receive bitcoin, and then end up with outbound liquidity from the bitcoin they received.
+
+One way to help your users receive inbound liquidity is through LSPs (Lightning Service Providers). You could run your own LSP service or integrate your product with an existing LSP.
+
+[Read the LSP section]({{'/guide/how-it-works/lightning-service-providers/#inbound-liquidity' | relative_url}}) to learn more. [Breez](https://breez.technology/) is an example of a wallet that does this.
+
+### Outbound liquidity
+
+There may be some situations where you need to help your user with outbound liquidity. An example of this would be a user who already owns bitcoin and wants to spend it using their lightning wallet.
+
+#### Automatic Channel Opens
```

Comment:
```suggestion
#### On-demand Channel Opens
```

This is the term used in the funding page, should mention why they are on-demand (opened when needed).

Reply:
I'm confused. When I read the [Funding page](https://deploy-preview-790--sad-borg-390916.netlify.app/guide/daily-spending-wallet/funding/#funding-the-wallet), it describes a funding flow wherein the user sends on-chain funds to their wallet to create a channel. Then the tip box about "On-demand Channels" says:

> Offering a on-demand channels Lightning service allows users to fund their wallet with a Lightning payment.

So the Funding page makes it seem like On-demand channels are something distinctly different, where the funding happens from a lightning payment and not an on-chain transaction.

I'm specifically talking about funding with an on-chain transaction and simply having the noff software on the phone automatically use those funds to initiate a channel open with another node. I feel like we are describing different things here.

Reply:
I need to re-work that page, but yeah on-demand channels allows users to fund their lightning wallet from another lighting wallet (or from an exchange) without it they have to send on-chain and conduct a swap / open a direct channel.

I made a mistake here, I referred to this type of open you are talking about as an [instant open](/guide/daily-spending-wallet/funding/#instant-open) not an on-demand channel open*